ganttproject/data/resources/language/i18n.properties

```diff
@@ -616,6 +616,7 @@ task.move.up = Up
 task.move.up.description = Move task up
 task.new = New task
 task.notesTooltip.pattern = <html>{0}</html>
+task.changeProgressHint = {0}% ({1}d)
```

Comment:
1.   `TimeUnitStack.encode` was a good alternative to another hardcoded "d" symbol
2.  Currently `TimeUnitStack.encode` is not i18-ed and it's not clear to me if it should  as some parts of code could potentially rely on that hardcoded "d" in logic. If it's not to be i18n-ed than we would have "d" symbol in hint for all the languages
3.   If  `TimeUnitStack.encode`needs i18n for its strings then we can do that and pass localized "d" as {2} in the discussed string. Or we can leave it as is considering hint text as independently localized string.